bittide/src/Bittide/ClockControl/Callisto.hs

```diff
@@ -179,3 +209,13 @@ callisto targetCount updateEveryNCycles mask allDataCounts =
   sign NoChange = 0
   sign SpeedUp = 1
   sign SlowDown = -1
+
+  rfCheck stable centered d cst@ControlSt{..}
+    | stable && not centered && not _rft =
+        cst { _css = d
+            , _rft = True
+            }
```

Comment:
For people not having worked on the algorithm a lot (everyone but you and Sanjay I suppose :-)), this is pretty hard to read. I'm getting lost in all the abbreviations `cst`, `css`, `rft`, `d`, ... Can they be expanded?

Reply:
Done.